api_tests/tests/utils.py

```diff
@@ -75,29 +74,29 @@ def send_consent_post(auth_token:str):
         return response
 
     @staticmethod
-    def send_consent_put(token:str, consent_id:str, version_id:str):
-        expected_status_code = 200
-
-        response = requests.put(
-            url=REASONABLE_ADJUSTMENTS_CONSENT + '/' + consent_id,
-            json=request_bank.get_body(Request.CONSENT_PUT),
+    def send_consent_get(auth_token: str):
+        response = requests.get(
+            url=REASONABLE_ADJUSTMENTS_CONSENT,
+            params={
+                'patient': '9692247317',
```

Comment:
This patient id is poping up in different places. Maybe get the id from method arguement or add this id to `config.py` so can be reused.